server/protocols/webadmin/webadmin-mailbox/src/main/java/org/apache/james/webadmin/service/ExportService.java

```diff
@@ -45,6 +45,25 @@
 
 public class ExportService {
 
+    public enum Stage {
+        STARTING,
+        ZIPPING,
+        EXPORTING,
+        COMPLETED,
+    }
+
+    public static class Progress {
+        private Stage stage;
```

Comment:
Use volatile keyword of an AtomicReference syntax

Reply:
Uninitialized variable will result in null, we should consider a `STARTING` as an initial value